### PR TITLE
[SD-1871] - Macros couldn't get loaded dynamically in celery

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: server/settings.py [hunk 1]
```diff
@@ -111,7 +111,11 @@ def env(variable, fallback_value=None):
     'spike:gc': {
         'task': 'apps.archive.content_purge',
         'schedule': crontab(minute=30)
-    }
+    },
+    'macros:print': {
+        'task': 'apps.macros.print_macros',
+        'schedule': timedelta(seconds=30),
```
Comment (amagdas) [inline server/settings.py]: I think the delta should be bigger, ~10 minutes, as this is not going to change really often.

**Comment by amagdas** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

File: server/settings.py (hunk 1 above)
Comment (petrjasek) [inline server/settings.py]: this part should be removed, it was there only for testing

**Pull request merged**